sandbox.go

```diff
@@ -691,6 +691,9 @@ func (sb *sandbox) EnableService() error {
 func (sb *sandbox) DisableService() error {
 	logrus.Debugf("DisableService %s START", sb.containerID)
 	for _, ep := range sb.getConnectedEndpoints() {
+		if err := ep.deleteServiceInfoFromCluster(sb, "DisableService"); err != nil {
```

Comment:
@abhinandanpb `serviceEnabled` is set before the `addServiceInfoToCluster` call and its reset if the call fails. Can you clarify what the problem in the current code is ?